lib/node_modules/@stdlib/math/base/complex/flipsign/docs/repl.txt

```diff
@@ -0,0 +1,39 @@
+
+{{alias}}( [out,] re, im, y )
+    Returns a complex number with the same magnitude as `z` and
```

Comment:
This same description can be applied elsewhere.

Reply:
...that is if you believe that the description is appropriate. I suppose it depends on whether it is fair to say the "sign" of a complex number. Do you have strong opinions here?

Reply:
I think that the `complex/signum` might serve as the "sign" in this case. I can't seem to think of a corner-case.

Reply:
In which case, I think it should be fine to use a description consistent with `special/flipsign`.

Reply:
So yes, it does make sense if we assume the sign of a complex number to be the signum function.